projects/gnu-tools/monthly-report-2020-12-14.md

```diff
@@ -0,0 +1,79 @@
+* Updated documentation to include all CORE-V extensions (found at: https://github.com/openhwgroup/core-v-docs/tree/master/cores/cv32e40p/user_manual)
+
+### Task status
+
+| Category    | Last Month | This Month | Delta    |
+| -----------:| ----------:| ----------:| --------:|
+| Unstarted   | 11         | 8          | -3       |
+| In progress | 3          | 3          | -        |
+| Complete    | 6          | 11         | +5       |
+
+In progress:
+* Add PI & RI L/S tests to binutils-gdb
+* Add db support to binutils-gdb
+* Project management - ongoing (new)
+
+Completed:
+* Update documentation to include CORE-V (new)
+* Add PI & RI L/S support to binutils-gdb
+* Add gen ALU ops support to binutils-gdb
+* Add gen ALU ops tests to binutils-gdb
+* Add gen ALU ops support to GCC
+
+### Testing status
+
+**GNU Binutils**
+
+GAS
+
+| Category    | This Month | Last Month   | Delta      | Baseline   | Delta      |
+| -----------:| ---------: | ---------:   | ---------: | ---------: | ---------: |
+| PASS        | 338        | 394          | 56         | 288        | 106        |
+| FAIL        | -          | -            | -          | -          | -          |
+| XPASS       | -          | -            | -          | -          | -          |
+| XFAIL       | 15         | 15           | -          | 15         | -          |
+| UNSUPPORTED | 10         | 10           | -          | 10         | -          |
+| UNTESTED    | -          | -            | -          | -          | -          |
+| UNRESOLVED  | -          | -            | -          | -          | -          |
+
+LD
+
+| Category    | This Month | Last Month   | Delta      | Baseline   | Delta      |
+| -----------:| ---------: | ---------:   | ---------: | ---------: | ---------: |
+| PASS        | 361        | 361          | -          | 357        | 4          |
+| FAIL        | -          | -            | -          | -          | -          |
+| XPASS       | -          | -            | -          | -          | -          |
+| XFAIL       | 4          | 4            | -          | 4          | -          |
+| UNSUPPORTED | 171        | 171          | -          | 171        | -          |
+| UNTESTED    | 26         | 26           | -          | 26         | -          |
+| UNRESOLVED  | -          | -            | -          | -          | -          |
+
+**GNU GCC**
+
+* CORE-V hardware loops, multiply accumulate and general ALU operations awaiting hardware testing
+
+### Risk register changes
+
+*   Two new risks added:
+    * No suitable target to test against - low risk as awaiting delivery of hardware
+    * University of Bologna / ETH Zurich cannot get FSF assignment - low risk as mitigation plan in place
+    * Insufficient resources to continue work, 15 months of work predicted (8 months from Embecosm an 2 months from UOB/ETHZ provided) - high risk as 5 months work required
```

Comment:
Add changed risk around vendor specific relocations. Note that setting up the RISC-V psABI TG is in progress (under the toolchain and runtimes subcommittee).  Those OpenHW Group members who are also RISC-V members and interested are encouraged to get involved.